Userland/Libraries/LibGfx/ImageFormats/WebPLoaderLossyTables.h

```diff
@@ -68,30 +68,26 @@ enum IntraBlockMode : u8 {
     num_intra_bmodes
 };
 
-// clang-format off
 static TreeIndex constexpr BLOCK_MODE_TREE[2 * (num_intra_bmodes - 1)] = {
- -B_DC_PRED, 2,                   /* B_DC_PRED = "0" */
-  -B_TM_PRED, 4,                  /* B_TM_PRED = "10" */
-   -B_VE_PRED, 6,                 /* B_VE_PRED = "110" */
+    -B_DC_PRED, 2, /* B_DC_PRED = "0" */
+    -B_TM_PRED, 4, /* B_TM_PRED = "10" */
+    -B_VE_PRED, 6, /* B_VE_PRED = "110" */
     8, 12,
-     -B_HE_PRED, 10,              /* B_HE_PRED = "11100" */
-      -B_RD_PRED, -B_VR_PRED,     /* B_RD_PRED = "111010",
-                                     B_VR_PRED = "111011" */
-     -B_LD_PRED, 14,              /* B_LD_PRED = "111110" */
-       -B_VL_PRED, 16,            /* B_VL_PRED = "1111110" */
-         -B_HD_PRED, -B_HU_PRED   /* HD = "11111110",
-                                     HU = "11111111" */
+    -B_HE_PRED, 10,         /* B_HE_PRED = "11100" */
+    -B_RD_PRED, -B_VR_PRED, /* B_RD_PRED = "111010",
+                               B_VR_PRED = "111011" */
+    -B_LD_PRED, 14,         /* B_LD_PRED = "111110" */
+    -B_VL_PRED, 16,         /* B_VL_PRED = "1111110" */
+    -B_HD_PRED, -B_HU_PRED  /* HD = "11111110",
+                               HU = "11111111" */
 };
-// clang-format on
```

Comment:
these were intentional to match the tree-like indentation of the spec as linked.

Reply:
Oh really? That simultaneously makes sense and is gross, but I'll revert this.

Reply:
t'was a request by @nico . :man_shrugging: